components/map/Map.js

```diff
@@ -17,7 +19,7 @@ export default function Map({ users }) {
       scrollWheelZoom={true}
       maxBounds={boundsMap}
       maxBoundsViscosity={0.7}
-      style={{ height: "100vh" }}
+      style={windowWidth > 1000 ? {height: "100vh"}:{height: "40vh"}}
```

Comment:
It doesn't work as expected right now. On resize, it saves previous height. I'd suggest adding state for window width, and wrap a `MapContainer `in `div`, which has height `100vh/40vh`, and MapContainer has `height: 100%`. Also, if it's necessary, you can move logic of getting window width to a separate hook.
````suggestion
  const [width, setWidth] = useState(window.innerWidth);

  useEffect(() => {
    const handleResize = () => setWidth(window.innerWidth);

    window.addEventListener("resize", handleResize);
    handleResize();

    return () => window.removeEventListener("resize", handleResize);
  }, []);

  const height = useMemo(() => (width > 1000 ? "100vh" : "40vh"), [width]);

  return (
    <div style={{ height }}>
      <MapContainer
        style={{ height: "100%" }}
      >
      ...
  );
```

Reply:
Thankyou for the suggestion

Reply:
I have updated the map.js file with the resize event